ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js

```diff
@@ -211,16 +219,14 @@ describe('component replace - Authoring', function () {
             cy.openAuthoring("/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure");
             updateAllowedComponent(true);
 
-            cy.openSiteAuthoring(replaceCompPageUrl);
+            cy.openSiteAuthoring(replaceCompPagePath);
             cy.selectLayer("Edit");
             cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Image", afConstants.components.forms.resourceType.formimage);
             cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + replaceCompEditPathSelector);
             cy.invokeEditableAction("[data-action='replace']");
             const replacementComp = "[value='" + replaceCompTestGroup + "']";
             cy.get(replacementComp).click();
             cy.deleteComponentByPath(replaceCompDrop);
-            cy.openSiteAuthoring("/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure");
```

Comment:
why this is removed? We should be revalidating after removal - by reloading the page.

Reply:
This is not validating anything, it only removes the test-group component from policies after validations are completed. However, I checked manually and cypress is clicking wrong element at this point.